src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
+                        _validators.TryGetValue(item.Category, out rule))
```

Comment:
what's the relationship between MarkdownTagValidationRule.fullid and MarkdownValidationRule.rulename?